tools/githooks/pre-commit

```diff
@@ -0,0 +1,30 @@
+#================================================================
+# HEADER
```

Comment:
I don't know what this #% annotations are for. Maybe just use normal shell comments?

Reply:
Done